pkg/sfu/receiver.go

```diff
@@ -122,8 +122,8 @@ type WebRTCReceiver struct {
 
 	onStatsUpdate        func(w *WebRTCReceiver, stat *livekit.AnalyticsStat)
 	onMaxLayerChange     func(maxLayer int32)
-	downtrackEverAdded   atomic.Bool
-	onDowntrackEverAdded func()
+	downTrackEverAdded   atomic.Bool
+	onDownTrackEverAdded func()
```

Comment:
some cosmetic changes here as we have been using `downTrack` with upper case `T`.